### ISSUE TITLE
Cannot retrieve the value when title is set with dynamic data

### ISSUE THREAD

**Issue by sharkx** [snapappointments/bootstrap-select#2877]
JSFiddle : <https://jsfiddle.net/stefanu/np8e1uwt/16/>

When the page is loaded, the `option`s are updated, the desired value is set, but when I try to retrieve it later (demonstrated by a delay), it is lost.

The problem seems to be the `title` attribute (set either as `title`, `data-title` or by script); to make it easier to play with the fiddle I left the script version. See line 10 of the script.

Simply put, when there's a `title` set, the value is lost... And I kind of need the title. Am I doing something wrong or this is really an issue ?

**Comment by whoshallsucceed** [comment]
It happens because you call `destroy`
